test/decl/var/variables.swift

```diff
@@ -14,10 +14,6 @@ var bfx : Int, bfy : Int
 
 _ = 10
 
-func _(_ x: Int) {} // expected-error {{keyword '_' cannot be used as an identifier here}}
```

Comment:
I moved it to test/Parse/identifiers.swift where I added a number of related test cases.